data/scripts.js

```diff
@@ -781,13 +781,7 @@ exports.BattleScripts = {
 			}
 
 			// Random ability
-			let abilities = [template.abilities['0']];
-			if (template.abilities['1']) {
-				abilities.push(template.abilities['1']);
-			}
-			if (template.abilities['H']) {
-				abilities.push(template.abilities['H']);
-			}
+			let abilities = Object.values(template.abilities);
```

Comment:
I guess the others aren't inner loops, so the readability is probably a good thing...